tests/baselines/reference/staticAsIdentifier.errors.txt

```diff
@@ -1,10 +1,10 @@
-tests/cases/compiler/staticAsIdentifier.ts(2,5): error TS1071: 'static' modifier cannot appear on an index signature.
+tests/cases/compiler/staticAsIdentifier.ts(2,12): error TS1030: 'static' modifier already seen.

```

Comment:
No, I think it's worth discussing whether this is a regression. Shipping runtimes already accept

```js
class Bar {
  static static
}
```

Reply:
The regression point is there, but there are several existing tests that already have `static static`:
* `cases/compiler/multipleClassPropertyModifiersErrors.ts`
* `cases/compiler/staticModifierAlreadySeen.ts`
* `cases/conformance/parser/ecmascript5/IndexMemberDeclarations/parserIndexMemberDeclaration10.ts`
* `cases/conformance/parser/ecmascript5/MemberVariableDeclarations/parserMemberVariableDeclaration2.ts`

Reply:
There is a difference in that `static static foo` on the same line doesn't exist today in ECMAScript.

Because there's a line terminator in this test, `static static` should continue to be whatever it is in ECMAScript due to automatic semicolon insertion rules.

In fact, I noticed that there's a bigger problem here which is that we currently error on code like

```ts
class C {
    static static
    foo() {}
}
```

which is actually totally acceptable JS. As a result, I've opened https://github.com/microsoft/TypeScript/issues/38283

Reply:
@DanielRosenwasser, so this comment can be resolved given the separate issue?